### ISSUE TITLE
Newest localstack command doesn't bind host to EGDE_PORT

### ISSUE THREAD

**Issue by danielrich** [localstack/localstack#2416]
# Type of request: This is a ...

[x ] bug report
[ ] feature request

# Detailed description
The edge port is not getting bound to the host in 0.11.1 or 0.11.1.1. It does get bound in 0.11.0.5. 

When you run 0.11.1.1 and run the 'localstack start' command and then examine the running container you see the ports "0.0.0.0:4567-4620->4567-4620/tcp, 0.0.0.0:8080-8081->8080-8081/tcp, 4566/tcp, 0.0.0.0:10000->10000/tcp"

and on 0.11.0.5 you will see the ports. "0.0.0.0:4566-4620->4566-4620/tcp, 0.0.0.0:8080-8081->8080-8081/tcp, 0.0.0.0:10000->10000/tcp"

notably on 0.11.1.1 the 4566 port is not bound to the host and thus is unaccessible outside of docker

## Expected behavior
The localstack commmand should make sure EDGE_PORT is bound to the host

## Actual behavior
The port is exposed in the docker container but not bound so it isn't available 

# Steps to reproduce

## Command used to start LocalStack
localstack start
docker ps (then examine ports)

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
no client code need

**Comment by afdecastro879** [comment]
I'm running from docker version 0.11.0 and is not binding the 4566 port either. This is breaking all my tests :'(

My env variables:
```
SERVICES=sqs,s3,es,elasticsearch
SQS_PORT_EXTERNAL=9324
SQS_PORT=9324
ELASTICSEARCH_PORT=9200
IMAGE_NAME=localstack/localstack:0.11.0
```

The process shows it bounds the port 4566

![Screenshot 2020-05-26 at 11 35 51](https://user-images.githubusercontent.com/5761473/82926746-422c9800-9f45-11ea-86ea-2a64dcb1a1f6.png)

However when running `docker ps`:

![Screenshot 2020-05-26 at 11 38 15](https://user-images.githubusercontent.com/5761473/82926851-68523800-9f45-11ea-8147-60ae0f1be17b.png)

**Comment by whummer** [comment]
Thanks for reporting @danielrich @afdecastro879 . This should be fixed in #2451 - can you please do a `pip install --upgrade localstack` and give it a try with the latest version? Please report here if the problem persists. Thanks
